crates/cli-support/src/wit/outgoing.rs

```diff
@@ -105,7 +105,7 @@ impl InstructionBuilder<'_, '_> {
                 // ... then defer a call to `free` to happen later
                 let free = self.cx.free()?;
                 self.instructions.push(InstructionData {
-                    instr: Instruction::DeferCallCore(free),
+                    instr: Instruction::DeferFree { free, align: 1 },
```

Comment:
I believe an alignment of 1 here is correct?

Reply:
Actually I don't think it is, because even though this is a String, it's taking out the bytes from a `Int32Array` memory.
Maybe it shouldn't be a `Int32Array` in the first place? But that's an issue for another time.

Reply:
> Actually I don't think it is, because even though this is a String, it's taking out the bytes from a `Int32Array` memory.

Are you sure? I think you might be looking at the code that fetches the ptr/len of the string from memory.